packages/elastic-sip-trunking/src/rest/v1/access-control-list/access-control-list-api.ts

```diff
@@ -0,0 +1,335 @@
+      'Accept': 'application/json',
+    };
+
+    const body: RequestBody = '';
+    const basePathUrl = `${this.client.apiClientOptions.hostname}/v1/projects/${this.client.apiClientOptions.projectId}/accessControlLists/${data['id']}`;
+
+    const requestOptions
+      = await this.client.prepareOptions(basePathUrl, 'DELETE', getParams, headers, body || undefined);
+    const url = this.client.prepareUrl(requestOptions.hostname, requestOptions.queryParams);
+
+    return this.client.processCall<void>({
+      url,
+      requestOptions,
+      apiName: this.apiName,
+      operationId: 'DeleteAccessControlList',
+    });
+  }
+
+  /**
+   * Delete IP range from ACL
+   * Remove an IP range to an access control list entry.
+   * @param { DeleteIpRangeFromAccessControlListRequestData } data - The data to provide to the API call.
+   */
+  public async deleteIpRange(data: DeleteIpRangeFromAccessControlListRequestData): Promise<void> {
+    this.client = this.getSinchClient();
+    const getParams = this.client.extractQueryParams<DeleteIpRangeFromAccessControlListRequestData>(
+      data,
+      [] as never[],
+    );
+    const headers: { [key: string]: string | undefined } = {
+      'Content-Type': 'application/json',
+      'Accept': 'application/json',
+    };
+
+    const body: RequestBody = '';
+    const basePathUrl = `${this.client.apiClientOptions.hostname}/v1/projects/${this.client.apiClientOptions.projectId}/accessControlLists/${data['accessControlListId']}/ipRanges/${data['ipRangeId']}`;
+
+    const requestOptions
+      = await this.client.prepareOptions(basePathUrl, 'DELETE', getParams, headers, body || undefined);
+    const url = this.client.prepareUrl(requestOptions.hostname, requestOptions.queryParams);
+
+    return this.client.processCall<void>({
+      url,
+      requestOptions,
+      apiName: this.apiName,
+      operationId: 'DeleteIpRangeFromAccessControlList',
+    });
+  }
+
+  /**
+   * List ACLs
+   * Fetches the list of Access Control List entries.
+   * @param { ListAccessControlListRequestData } data - The data to provide to the API call.
+   * @return { ApiListPromise<AccessControlList> }
+   */
+  public list(data: ListAccessControlListRequestData): ApiListPromise<AccessControlList> {
+    this.client = this.getSinchClient();
+    const getParams = this.client.extractQueryParams<ListAccessControlListRequestData>(data, [] as never[]);
+    const headers: { [key: string]: string | undefined } = {
+      'Content-Type': 'application/json',
+      'Accept': 'application/json',
+    };
+
+    const body: RequestBody = '';
+    const basePathUrl = `${this.client.apiClientOptions.hostname}/v1/projects/${this.client.apiClientOptions.projectId}/accessControlLists`;
+
+    const requestOptionsPromise = this.client.prepareOptions(basePathUrl, 'GET', getParams, headers, body || undefined);
+
+    const operationProperties: PaginatedApiProperties = {
+      pagination: PaginationEnum.PAGE2,
```

Comment:
Not related to this PR (see comment from another PR related to pagination handling from client/core)
`PaginationEnum.PAGE2`, `PAGE3`, ... are not very intuitive

Reply:
Totally agree: at the beginning, there was only TOKEN and PAGE, but then I noticed that the parameters for the PAGE pagination are different from one API to the other and even the start page is not the same. So as Fax is v3, I thought the APIs were following a kind of evolution and I created a new pagination type called PAGE3.
And then I saw this one and I was out of ideas to find a name that would characterise the pagination behaviour. So I went for the opportunistic PAGE2 that was conveniently filling the gap 😅